### DISCUSSION TITLE
造成轟動的語言模型 chatgpt

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#894] [General]
讓我引用我在別的地方看到的一段話，chatgpt 在自然語言處理這塊可以說是劃時代的里程碑，他優於九成專用於特定領域的語言模型，且不限於英語橫跨許多種不同的語言。
如果想知道更多可以點以下連結!
https://www.bbc.com/news/technology-63861322

**Comment by Chiway1229** [comment]
ChatGPT真的非常有趣，上次上課時就玩得不亦樂乎

**Comment by bonny123865** [comment]
chatGPT真的很強!
上課的時候有用過，而且跟他聊天還會連接上問繼續回復
不會單指目前這段話做回覆，超讚的
而且問他有關程式的東西還會直接寫出一個範例給我看 超讚

**Comment by Yunhans** [comment]
不過有時候給的答案都怪怪的，還是得自己google找答案

**Comment by ching020721** [comment]
連其他課的老師都有跟我們展示chatgpt的回覆很方便，可以當作小助手，不會的問題問他就對了

**Comment by yuxinlin93** [comment]
chatGTP 真的很有趣，看到這篇文章我也去玩玩看，我請他講幾個冷笑話給我聽，但他對於冷笑話的口味非常非常的特別，特別到讓我有點心理發寒，也許開發chatGPT 的開發人員可以多餵一些冷笑話的資源進去

**Comment by Floriaand** [comment]
我覺得真的對學習很有幫助，問它數學它居然也回答得出來，要是高三的時候就有這個東西就好了

**Comment by HuangHsinTzu** [comment]
chatGPT 真的超酷 比Siri強太多了
不僅是聊天 給意見 還可以幫你解決問題!
之前有朋友問他有什麼推薦的電影
它居然就列了一長串電影清單
雖然chatGPT真的很方便可以幫我們解決許多問題
但我個人還是覺得它的答案參考參考就好
還是自己去做確認的動作
畢竟研發公司自己都說他們不能保證chatGPT不會給出帶有歧視或有問題的回答了

**Comment by evanchou1204** [comment]
chatGPT真的很厲害，跟以前看過的聊天機器人差很多，講出來的話明顯是有邏輯的回答，太強了。

**Comment by orianna901001** [comment]
我雖然上次用chatgpt在尋找推薦的開放式課程跟推薦論文時，它給出了根本不存在的網址(當下我以為是我電腦的問題)，整體來說，雖然文章看起來非常有架構性，但在這方面還有待加強。
